projects/components/src/action-menu/action-menu.component.html

```diff
@@ -0,0 +1,80 @@
+<div class="inline-actions-container" *ngIf="shouldDisplayActions(actionStyling.INLINE)">
```

Comment:
Is there other cases where `shouldDisplayActions` is called with other(different) parameters? If not - then why not using a property that is calculated just once